httpie/client.py

```diff
@@ -42,28 +43,68 @@ def init_poolmanager(self, *args, **kwargs):
         super(HTTPieHTTPAdapter, self).init_poolmanager(*args, **kwargs)
 
 
-def get_requests_session(ssl_version):
-    requests_session = requests.Session()
-    requests_session.mount(
-        'https://',
-        HTTPieHTTPAdapter(ssl_version=ssl_version)
-    )
-    for cls in plugin_manager.get_transport_plugins():
-        transport_plugin = cls()
-        requests_session.mount(prefix=transport_plugin.prefix,
-                               adapter=transport_plugin.get_adapter())
-    return requests_session
+REMOVE_BODY = 0
+KEEP_BODY = codes.temporary_redirect
+
+
+class HTTPieRequestsSession(requests.Session):
+
```

Comment:
This is pretty much all kludge and I hate myself for doing it but see no other way without getting changes accepted into `requests` first.